src/events.c

```diff
@@ -41,6 +45,35 @@ static void inform_keyboard_attached(gboolean value)
         }
 }
 
+static void dir_changed_cb(GFileMonitor *monitor,
+               GFile *file, GFile *other_file,
+               GFileMonitorEvent event_type, gpointer user_data) {
```

Comment:
I'd indent it like this:
```
static void dir_changed_cb(GFileMonitor *monitor,
                           GFile *file,
                           GFile *other_file,
                           GFileMonitorEvent event_type,
                           gpointer user_data)
{
```

Reply:
I'm inclined to keep it the way it is, unless you have strong objections.

Reply:
Ok, will fix.